source/core/chat_bot_template.py

```diff
@@ -17,14 +17,31 @@
 
 logger = logging.getLogger(__name__)
 
+MAIN, GRAPH = range(2)
```

Comment:
Use Python class Enum to introduce constant state variables

Reply:
Okay, I'll fix it in next commit.